### DISCUSSION TITLE
前一天刷的课程，第二天清空了

### DISCUSSION THREAD

**Discussion by Ailontania** [liuyunfz/chaoxing_tool#101] [General]
前一天刷的课程，第二天清空了
使用的2.1版本，选项1，第二天被清空了。

**Comment by liuyunfz** [comment]
详见https://github.com/liuyunfz/chaoxing_tool/issues/32
猜测是超星平台的全局检测，对多个任务点同时完成的一种反馈检测
暂时没有解决办法，只能继续重复刷
